pkg/controller/util/node/controller_utils.go

```diff
@@ -171,36 +153,6 @@ func MarkAllPodsNotReady(kubeClient clientset.Interface, node *v1.Node) error {
 	return fmt.Errorf("%v", strings.Join(errMsg, "; "))
 }
 
-// ExistsInCloudProvider returns true if the node exists in the
-// cloud provider.
-func ExistsInCloudProvider(cloud cloudprovider.Interface, nodeName types.NodeName) (bool, error) {
```

Comment:
replaced by [ensureNodeExistsByProviderID](https://github.com/kubernetes/kubernetes/blob/521028ed52526abd832f8a6c9c5ec62340fccbb2/pkg/controller/cloud/node_controller.go#L446-L467) which is functionally the same but uses providerID if it exists, otherwise uses `InstanceID`.